src/Fixer/ClassNotation/FinalInternalClassFixer.php

```diff
@@ -157,23 +162,26 @@ protected function createConfigurationDefinition()
      */
     private function isClassCandidate(Tokens $tokens, $index)
     {
-        if ($tokens[$tokens->getPrevMeaningfulToken($index)]->isGivenKind([T_ABSTRACT, T_FINAL])) {
+        if ($tokens[$tokens->getPrevMeaningfulToken($index)]->isGivenKind([T_ABSTRACT, T_FINAL, T_NEW])) {
             return false; // ignore class; it is abstract or already final
         }
 
         $docToken = $tokens[$tokens->getPrevNonWhitespace($index)];
 
         if (!$docToken->isGivenKind(T_DOC_COMMENT)) {
-            return false; // ignore class; it has no class-level PHPDoc
+            return $this->configuration['consider-absent-docblock-as-internal-class'];
         }
 
         $doc = new DocBlock($docToken->getContent());
         $tags = [];
 
         foreach ($doc->getAnnotations() as $annotation) {
-            $tag = strtolower($annotation->getTag()->getName());
-            if (isset($this->configuration['annotation-black-list'][$tag])) {
-                return false; // ignore class: class-level PHPDoc contains tag that has been black listed through configuration
+            Preg::match('/@\S+(?=\s|$)/', $annotation->getContent(), $matches);
```

Comment:
A lot of online doc about Doctrine refers to entities with the tag `@ORM\Entity`, which is useful also because you have to write only one `use` for everything Doctrine-related an entity needs.
But the backslash breaks what is considered a tag in this regex:

https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/110bc117adbbda23ac7770a670aa95947123476e/src/DocBlock/Tag.php#L70

Without this change `@ORM\Entity` can be handled properly: `other` would be too generic IMHO.